docs/examples/add-to-your-application/ruby/data_filtering.rb

```diff
@@ -0,0 +1,93 @@
+require 'oso'
+require 'sequel'
+require 'sinatra'
+
+require_relative './models'
+
+DB = Sequel.sqlite
+
+DB.create_table :repositories do
+  String :name
+  TrueClass :is_public
+end
+
+repositories = DB[:repositories]
+
+module DF
+class Repository < Sequel::Model(:repositories)
+end
+
+r = Repository.new(name: "gmail", is_public: false)
+r.save
+
+OSO = Oso.new
```

Comment:
I think you could just use an `oso` variable (not an `OSO` constant) to hold the oso instance.
```suggestion
oso = Oso.new
```

Reply:
I don't think it's accessible on L82 then.